scripts/ci/linux/setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+echo "Configuring build type '$BUILD_TYPE'"
+mkdir build
+
+conda update -n base -c defaults conda
+conda install cmake ninja conda-build compilers -y
```

Comment:
Is `conda-build` actually required?

Reply:
conda-build gets you compilers, cmake, and ninja. Since they are explicitly stated, we don't need it.

Reply:
conda-build removed.